apps/style/common.scss

```diff
@@ -390,6 +390,21 @@ html[dir="rtl"] .editor-column {
   background-position: 2px 2px;
 }
 
+html[dir="rtl"] {
+  // !Important is used to overwrite droplet.min styling for RTL languages support.
+  #codeTextbox {
+    background-color: red;
```

Comment:
do we actually want the background to be red, or is this test code?

Reply:
My mistake. Thanks a lot for noticing!